frontend/packages/kubevirt-plugin/src/selectors/vm-template/advanced.ts

```diff
@@ -49,6 +56,14 @@ export const getTemplatesLabelValues = (templates: TemplateKind[], label: string
 export const getTemplateFlavors = (templates: TemplateKind[]) =>
   getTemplatesLabelValues(templates, TEMPLATE_FLAVOR_LABEL);
 
+export const getTemplateHostname = (template: TemplateKind) => {
```

Comment:
btw, shouldn't we first check for the YAML hostname and start fetching it from the cloudinit if there is none?